db/model/model.js

```diff
@@ -142,3 +127,28 @@ exports.fetchAllUsers = () => {
     )
     .then(({ rows }) => rows);
 };
+exports.getArticlesByTopic = (topic = "PassBYChecker") => {
+  let validTopicQueries = ["mitch", "cats", "PassBYChecker"];
+  if (!validTopicQueries.includes(topic)) {
+    return Promise.reject({ status: 400, msg: "Invalid sort_by query" });
+  }
+  validTopicQueries = ["mitch", "cats"];
```

Comment:
This should be sorted via the db not hardcoded as your production data will be different. Please check if topic exists another way instead